src/vs/workbench/contrib/debug/browser/breakpointsView.ts

```diff
@@ -622,6 +627,10 @@ class DataBreakpointsRenderer implements IListRenderer<DataBreakpoint, IBaseBrea
 		if (session && !session.capabilities.supportsDataBreakpoints) {
 			data.breakpoint.title = localize('dataBreakpointsNotSupported', "Data breakpoints are not supported by this debug type");
 		}
+		if (dataBreakpoint.accessType) {
+			const accessType = dataBreakpoint.accessType === 'read' ? localize('read', "Read") : dataBreakpoint.accessType === 'write' ? localize('write', "Write") : localize('access', "Access");
+			data.accessType.textContent = accessType;
+		}
```

Comment:
We need the `else` case here
`data.accessType.textContent = ''`
Reason is that these templates are reused between different elements, so we need to clear out the field if the data breakpoint does not have an `accessType`

Reply:
Done in last commit